R/tags.R

```diff
@@ -879,11 +887,14 @@ flattenTags <- function(x) {
       unlist(lapply(x, flattenTags), recursive = FALSE)
     }
 
-  } else if (is.character(x)){
+  } else if (is.character(x)) {
     # This will preserve attributes if x is a character with attribute,
     # like what HTML() produces
     list(x)
 
+  } else if (is_html_dependency(x)) {
+    list()
```

Comment:
Needed to prevent infinite recursion now that `as.tags.html_dependency()` returns a `tagList()` of length 1